modules/notification/action/action.go

```diff
@@ -100,14 +101,15 @@ func (a *actionNotifier) NotifyCreateIssueComment(doer *user_model.User, repo *m
 		IsPrivate: issue.Repo.IsPrivate,
 	}
 
-	content := ""
-
-	if len(comment.Content) > 200 {
-		content = comment.Content[:strings.LastIndex(comment.Content[0:200], " ")] + "…"
-	} else {
-		content = comment.Content
+	truncatedContent, truncatedRight := util.SplitStringAtRuneN(comment.Content, 200)
```

Comment:
I think this one is intended to be bytes not runes

Reply:
I was thinking about `varchar(10)` is rune-based. 😂

Actually using `SplitStringAtByteN` seems better and the database column is `TEXT`.

Updated.